lib/thredded/truthy_hash.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Thredded
+  module TruthyHash
```

Comment:
@glebm good point about changing the default. Given that the default might change according to admin config whim, it might be better to just record all the values agreed to by the user ie. some form of `Hash<string, boolean>` in the db (serialized as whatever) and then let the default be part of configuration (into which it is marshalled)

The default can only be either true or false which can be part of the notifier set up:

```ruby
Thredded.notifiers = [
  EmailNotifier.new(defaults: {notifications_for_followed_topics: true, messageboard_notifications_for_followed_topics: true, notifications_for_followed_topics: true})
]
```

For clarity the notifier should require that all defaults are specified if any are specified.

Any further thoughts?